openmetadata-service/src/main/resources/elasticsearch/en/test_case_index_mapping.json

```diff
@@ -152,20 +152,14 @@
           "description": {
             "type": "text"
           },
-          "entityType": {
+          "type": {
             "type": "keyword"
-          },
-          "testPlatforms": {
-            "type": "text",
-            "fields": {
-              "keyword": {
-                "type": "keyword",
-                "ignore_above": 256
-              }
-            }
```

Comment:
`testDefinition` is an entityReference and does not have `entityType` or `testPlatforms`